_posts/2023-06-26-trino-fest-2023-salesforce.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Trino Fest 2023 recap: Anomaly detection for Salesforce’s production data using Trino"
+author: "Tuli Nivas, Geeta Shankar, and Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Salesforce.png
+show_pagenav: false
+---
+
+Rolling into our next presentation of Trinofest 2023, we're excited to bring you
+Tuli Navas and Geeta Shankar's talk from the Performance Engineering Team of
+Salesforce. They provide numerous reasons for why they need Trino and
+further continue explaining how it is essential for anomaly detection within
+their data. It's an insightful talk of how expeditious Trino can be that you definitely
```

Comment:
"how expeditious Trino can be" is pretty awkward wording, and that's not the core of their talk. It's _part_ of it, but you might want to reframe this. "It's an insightful talk about using a query engine to ensure data quality, and you get to hear about how switching to Trino delivered massive performance improvements." or something along those lines reads better and is closer to the truth about the thesis of their talk.